docs/pages/includes/database-access/start-auth-proxy.mdx

```diff
@@ -1,27 +1,41 @@
+Teleport requires a valid TLS certificate to operate and can fetch one automatically
+using Let's Encrypt's [ACME](https://letsencrypt.org/how-it-works/) protocol. 
+
+<Tabs>
+<TabItem scope={["oss","enterprise"]} label="Self Hosted">
+We will assume that you have configured DNS records for `teleport.example.com` and
+`*.teleport.example.com` to point to your Teleport node.
+
+<Admonition type="note" title="Web Proxy Port">
+  The ACME protocol must verify that an HTTPS server is reachable on port 443 of a host before Let's Encrypt can issue a TLS certificate with that host's domain as a subject. Port 443 must be open to all incoming traffic on the Teleport Proxy host.
```

Comment:
> Port 443 must be open to all incoming traffic on the Teleport Proxy host.

This wording may be a little confusing, I would say that Teleport Proxy itself needs to listen on 443 (as opposed to default 3080), not just that 443 must be open.

Reply:
I've changed the information in the Admonition box per this comment. To make the "Self Hosted" tab more cohesive, I've removed the link re: the ACME protocol and added the relevant information to the first paragraph.